wayland-client/src/globals.rs

```diff
@@ -243,8 +237,8 @@ impl std::error::Error for BindError {}
 impl fmt::Display for BindError {
     fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
         match self {
-            BindError::UnsupportedVersion {} => {
-                write!(f, "the requested version of the global is not supported")
+            BindError::UnsupportedVersion(version) => {
+                write!(f, "the requested version {version} of the global is not supported")
```

Comment:
This wording suggests that `{version}` is the version that was requested by the user, not that advertised by the compositor.

Reply:
Ah my mistake, I got confused there!
Should I remove the `{version}` completely or reorder the sentence to announce the actually available version?

Reply:
Maybe change the sentence so that it uses `version` but in a meaningful way

Reply:
What do you think of `the available version {version} of the global is lower than the requested version`?

Reply:
sound good yes :+1: